frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -264,6 +264,9 @@ func (f *fetchCVDCommandArtifactsFetcher) Fetch(outDir, buildID, target string,
 			fetchCmd.Args = append(fetchCmd.Args, "--credential_source=gce")
 		}
 	}
+	if f.buildAPICredentials.CredsUserProjectID != "" {
+		fetchCmd.Args = append(fetchCmd.Args, "--project_id="+f.buildAPICredentials.CredsUserProjectID)
+	}
```

Comment:
Likewise, should we set `--project_id` only when `f.buildAPICredentials.AccessToken != ""`?